### ISSUE TITLE
Import external data into cell?

### ISSUE THREAD

**Issue by calissendorff** [andmarti1424/sc-im#607]
Google Sheets has something called ImportXML which lets you import various types of data from a website url into a cell.

Does sc-im have anything similar?

Or alternatively, given that the file format seems quite simple, what would be a good way of replacing a number in a certain cell with data from the internet?

For instance, replacing:

`let C2 = 0.75`

... with ...

`let C2 = 0.81`

... getting the number from some data on the web (perhaps thru a GET or POST api call)

**Comment by andmarti1424** [comment]
No. But you can use lua scripts for achiveing the same thing. Check examples in `examples/lua` folder.

**Comment by calissendorff** [comment]
Can you give me a clue how to do this? I placed `getstock.lua` in `~/.sc-im/lua/` and typed `=@lua("getstock.lua", 1)` in sc-im and I get a segmentation fault.

**Comment by andmarti1424** [comment]
First, it should not segfault. If you can, please share stacktrace or valgrind log.
Segundo, try placing it in `$HOME/.config/sc-im/lua/`
You also have to enter the `@lua` functions as string functions, with `\"`
